include/deal.II/fe/fe.h

```diff
@@ -1356,11 +1356,28 @@ class FiniteElement : public Subscriptor, public FiniteElementData<dim>
    * For a definition of domination, see FiniteElementDomination::Domination
    * and in particular the
    * @ref hp_paper "hp paper".
+   *
+   * @deprecated Use compare_for_domination() with `codim=1` instead.
    */
-  virtual FiniteElementDomination::Domination
+  DEAL_II_DEPRECATED virtual FiniteElementDomination::Domination
   compare_for_face_domination(
     const FiniteElement<dim, spacedim> &fe_other) const;
 
+  /**
+   * Return whether this element dominates a neighboring one given as argument,
+   * whether it is the other way around, whether neither dominates, or if
+   * either could dominate. The @p codim parameter describes the codimension of
+   * of the investigated subspace and specifies that it is subject to this
+   * comparison.
```

Comment:
```suggestion
   * comparison. For example, if `codim==0` then this function compares which element dominates at the cell level.
   * If `codim==1`, then the elements are compared at faces, i.e., the comparison happens between the function
   * spaces of the two finite elements as restricted to a face. Larger values of `codim` work correspondingly.
```

Reply:
I tried a new feature of github here whereby I make a concrete code suggestion. See what happens with this comment and what you can do with it.

Reply:
Ah, I can commit your suggestion separately. I'll give it a try, since I'll squash those commits anyway.

Reply:
I'm sure Travis will complain about the line length of the comments :D

Reply:
It does :-)

Reply:
It'll be fixed when you squash.